apps/api/.env.example

```
@@ -9,4 +9,5 @@ GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
 FRONTEND_URL=http://localhost:3001
 OPENROUTER_API_KEY=your-key-here
-AI_MODEL=nvidia/nemotron-3-ultra-550b-a55b:free
\ No newline at end of file
+AI_MODEL=nvidia/nemotron-3-ultra-550b-a55b:free
+CONVERSATION_WINDOW_SIZE=5
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)